src/App.js

```diff
@@ -2,10 +2,12 @@ import React from "react";
 import { Route, Switch } from "react-router-dom";
 import { Home } from "./Home/Home";
 import { Header } from "./Header/Header";
+import {Footer} from "./zFooter/Footer";
```

Comment:
This (`zFooter/Footer`) really does look bad, and people, if they review your GitHub, will wonder why you've done this. 😄

Reply:
Ok, I'll fix it. I wish there was a way to keep the folders organized in an intuitive way. Footer being at the top of my file directory doesn't make me very happy 😒